tests/test_config_parsing/test_new_observations_parser.py

```diff
@@ -68,3 +74,44 @@ def test_parse():
             ),
         ]
     )
+
+
+def test_parse_include_outside_raises_obsconf_error():
```

Comment:
I am trying to think about a better name for the test as this is not clear to me at a first glance.
Would you have a suggestion?

Reply:
Tried to rename it to `test_that_unexpected_character_gives_observation_config_error`, maybe it is a bit more clear

Reply:
It is much better. Thanks 🚀